implementation/src/main/java/io/smallrye/config/mapper/ConfigMapping.java

```diff
@@ -94,6 +94,10 @@ static String skewer(String camelHumps) {
 
     static String skewer(String camelHumps, int start, int end, StringBuilder b) {
         assert !camelHumps.isEmpty() : "Method seems to have an empty name";
+        if (camelHumps.startsWith("get")) {
```

Comment:
I disagree with this for two reasons:

1. I don't like it :)  I think naming the properties exactly is the best, especially since you could potentially have a boolean property which is a verb: `get-file-size-before-uploading` or some such.
2. The implementation allows two properties with the same name to exist, which is inherently problematic and necessitates more validation.  Not really worth the headache to support this convention.

Reply:
It's fine, I'll remove it. My idea here is the you may already have a POJO that you could reuse.